global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "7.0.102"
```

Comment:
Building the .Net 6 packages using the .Net 7 SDK results in packages that cannot be used in a .Net 6 application.

Reply:
Done, I updated globa.json to target 6.0.407